{"status":"FAIL","msg":"FAIL:\n Uncaught: { message: \"Expected a Test262Error but got a TypeError (Testing with Function.)\", name: \"Test262Error\" }\n    at ../../test262/harness/testBigIntTypedArray.js:37\n    at testWithBigIntTypedArrayConstructors (../../test262/test/built-ins/TypedArrayConstructors/from/BigInt/arylk-get-length-error.js:24)\n\n","path":"../../test262/test/built-ins/TypedArrayConstructors/from/BigInt/arylk-get-length-error.js","memory_kb":0,"duration":623008831}